libs/blocks/text/text.css

```diff
@@ -83,12 +102,63 @@
   inset: 0 100% auto auto;
 }
 
+.text-block.accent-bar {
+  inline-size: 100%;
+  block-size: var(--accent-height);
+}
+
+.text-block .lockup-area img {
+  width: auto;
+}
+
 /* Alignment */
 .text-block.center {
   text-align: center;
   align-items: center;
 }
 
+.text-block .icon-area {
+  display: flex;
+  column-gap: var(--spacing-xs);
+}
+
+.text-block p.icon-area { /* NOT <a/> tags with icons in them */
+  margin-block-end: var(--spacing-m);
+}
+
+.text-block.xxs-icon .icon-area {
```

Comment:
Shouldn't these just use the iconograpy.css styles and these wouldn't be needed at all?

Reply:
@ryanmparrish I totally agree, this was just me correcting a `style-lint` error. I'd be happy to remove it, but I believe that will require some extra regression testing.

Reply:
Yes,  since you're introducing `iconography.css` in this PR. 
We should definitely trim the fat since this is redundant now.

Reply:
okay, I just ask because `iconography.css` does not include any rules related to `.icon-area` whatsoever. I do not import that stylesheet unless a block specifically uses a `lockup` variant, because otherwise we would be adding an HTTP request to every single use of the Text block.

Are you _sure_ we should be removing this?

Reply:
Wouldn't we then just change that selector to `.lockup-area` in the decorator and test it still works as expected. Its essentially just doubling up the styles w/ diff selectors doing the same thing.

Reply:
I agree moving forward that is probably the right idea, but like I said, this was a pre-existing style that I just moved to fix a style-lint error. Taking this rule out altogether runs the risk of messing up old usages of this block.

Reply:
Fair, there also might be a day they are different. Probs safer to just leave em be for now.